tests/system/providers/apache/pinot/example_pinot_dag.py

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Example DAG demonstrating the usage of the PinotAdminHook and PinotDbApiHook."""
+
+from __future__ import annotations
+
+from datetime import datetime
+
+from airflow import DAG
+from airflow.decorators import task
+from airflow.providers.apache.pinot.hooks.pinot import PinotAdminHook, PinotDbApiHook
+
+with DAG(
+    dag_id="example_pinot_hook",
+    schedule_interval=None,
+    start_date=datetime(2021, 1, 1),  # Override to match your needs
+    tags=["example"],
```

Comment:
```suggestion
    tags=["example"],
    catchup=False,
```
We should add `catchup=False` so there aren't a year+ of DAG Runs that get created when this system test runs if the `start_date` isn't updated. This is typical in the example DAGs throughout the repo too.